src/UI/UIUtils.java

```diff
@@ -175,6 +176,7 @@ public static ColorPalette getMapUnitColors(Color colorKey)
     {
       mapUnitColorPalettes.put(colorKey, mapUnitColorPalettes.get(Color.PINK));
       palette = mapUnitColorPalettes.get(colorKey);
+      System.out.println(String.format("WARNING!: Failed to retrieve unit palette for color %s, defaulting to PINK", colorKey));
```

Comment:
Instead of specifying PINK, maybe use `UIUtils.getColorName(Color.PINK)`. Magic strings, and all that.